### PR TITLE
Fix EBS CSI Driver Issue - Avoiding `CoreAddonProps` to be filled out 

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*
Fix EBS CSI Driver Issue - Avoiding `CoreAddonProps` to be filled out 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/addons/ebs-csi-driver/index.ts [hunk 1]
```diff
@@ -8,16 +8,18 @@ import { KubernetesVersion } from "aws-cdk-lib/aws-eks";
...
+    [KubernetesVersion.V1_28, "v1.28.0-eksbuild.1"],
+    [KubernetesVersion.V1_27, "v1.28.0-eksbuild.1"],
+    [KubernetesVersion.V1_26, "v1.28.0-eksbuild.1"]
 ]);
 
 /**
  * Interface for EBS CSI Driver EKS add-on options
  */
-export interface EbsCsiDriverAddOnProps extends CoreAddOnProps {
+export type EbsCsiDriverAddOnProps = Omit<CoreAddOnProps, "saName" | "addOnName" > & {
```
Comment (shapirov103) [inline lib/addons/ebs-csi-driver/index.ts:22]: sounds like you will be able to still pass  versionMap here. 
I think we should exclude all the following:

```
addOnName
saName
namespace
controlPlaneAddOn
versionMap
```

**Review by shapirov103 — COMMENTED** [review]
@elamaran11 please see my comment.

File: lib/addons/ebs-csi-driver/index.ts (hunk 1 above)
Comment (elamaran11) [reply lib/addons/ebs-csi-driver/index.ts]: @shapirov103 Fixed pls check.

**Review by shapirov103 — APPROVED** [review]
LGTM, TY!

**Pull request merged**